java/ql/integration-tests/java/query-suite/test.py

```diff
@@ -0,0 +1,15 @@
+import runs_on
+
+@runs_on.linux
+def test(codeql, java, cwd, expected_files, semmle_code_dir):
+    query_suites = ['java-code-quality.qls', 'java-security-and-quality.qls', 'java-security-extended.qls', 'java-code-scanning.qls']
+
+    for query_suite in query_suites:
+        actual = codeql.resolve.queries(query_suite, _capture=True).strip()
+        actual = sorted(actual.split('\n'))
+        print(semmle_code_dir)
+        index = len(str(semmle_code_dir))
+        actual = [line[index:] for line in actual]
```

Comment:
ah, if you use `semmle_code_dir` then you can just use `relative_to` with it, which has a slightly clearer intent. This will remove all leading `/` in the expected files (you can update the files locally with `--learn`):
```suggestion
        actual = [str(pathlib.Path(q).relative_to(semmle_code_dir)) for q in actual]
```

or alternatively
```suggestion
        actual = [os.path.relpath(q, semmle_code_dir) for q in actual]
```